zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/JobRequestValidator.java

```diff
@@ -52,4 +55,26 @@ public static Optional<ProblemDetail> validateJobErrorRequest(
     }
     return createProblemDetail(violations);
   }
+
+  public static Optional<ProblemDetail> validateJobUpdateRequest(
+      final JobUpdateRequest updateRequest) {
+    final List<String> violations = new ArrayList<>();
+    final JobChangeset changeset = updateRequest.getChangeset();
+    if (changeset.getRetries() == null && changeset.getTimeout() == null) {
+      violations.add(
+          ERROR_MESSAGE_AT_LEAST_ONE_FIELD.formatted(
+              String.join(", ", List.of("retries", "timeout"))));
```

Comment:
🔧 I believe it's sufficient to provide a list parameter, the String representation is `[retries, timeout]`, so the whole message will read "At least one of [retries, timeout] is required." in the end. Would that work or would you like the message to look differently?

Reply:
I agree to simplify this 👍